src/EMS/PatientFields.js

```diff
@@ -13,6 +13,19 @@ import Ringdown from '../Models/Ringdown';
 import ApiService from '../ApiService';
 import Context from '../Context';
 
+const INPUT_RANGES = {
+  systolicBloodPressure: {min: 120, max: 180},
+  diastolicBloodPressure: {min: 80 , max: 120},
```

Comment:
Let's set the min for systolic to be 90 and the min for diastolic to be 60... if a responder takes a measurement, and it's normal, I think they should still be able to record it...?